frontend/__tests__/sum.test.ts

```diff
@@ -0,0 +1,21 @@
+import { sum } from "../src/sum"
```

Comment:
this file should not be in the src/ directory either

Reply:
moved